builder/lxd/step_lxd_launch.go

```diff
@@ -17,22 +18,35 @@ func (s *stepLxdLaunch) Run(_ context.Context, state multistep.StateBag) multist
 
 	name := config.ContainerName
 	image := config.Image
+	profile := fmt.Sprintf("--profile=%s", config.Profile)
 
-	args := []string{
-		"launch", "--ephemeral=false", image, name,
+	launch_args := []string{
+		"launch", "--ephemeral=false", profile, image, name,
+	}
+
+	for k, v := range config.LaunchConfig {
+		launch_args = append(launch_args, fmt.Sprintf("--config %s=%s", k, v))
```

Comment:
This is broken. It should be
`launch_args = append(launch_args, "--config", fmt.Sprintf("%s=%s", k, v))`